python/ray/air/config.py

```diff
@@ -206,7 +206,7 @@ def _resources_per_worker_not_none(self):
 
         if self.accelerator_type:
             accelerator = f"{RESOURCE_CONSTRAINT_PREFIX}{self.accelerator_type}"
-            resources_per_worker.setdefault(accelerator, 1)
+            resources_per_worker.setdefault(accelerator, 0.001)
```

Comment:
Can we make this a constant (or use an existing one if it already exists)?

Reply:
https://github.com/ray-project/ray/blob/2747c80c189e4a5c9cdf7de853f4967ef3a93ea7/python/ray/_private/utils.py#L389-L392

Seems core team directly use 0.001 here.

Reply:
@jjyao cool if we extract this into a constant? Gives it some concrete meaning 🙂